portfolio/src/main/webapp/script.js

```diff
@@ -112,3 +112,44 @@ function selectFractal(idx) {
         document.querySelector('#fractal').alt = "";
     }
 }
+
+function createMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map_'),
+        {center: {lat: 37.42786582988798, lng: -122.17018205695274}, zoom: 15}
+    );
+  
+    addLandmark(
```

Comment:
There are a bunch of these function calls, and it would be quite hectic trying to maintain this if you had to add more or remove some. Basically, the information that you need to store for a location is longitude, latitude, name, and description and it would be nice if we could store this location data somewhere for all locations. There are a few different ways to address this.

1) You could store location data in your JavaScript code
2) You could set up a backend endpoint to your server called something like `/locations` where your locations are stored as static array of locations in your server code, and you make a call to this backend endpoint from your JavaScript code. You would have to convert your Location objects to JSON (ping me if you need help!)
3) You could store all of your locations in a database, and make a call to your backend server which then calls your database.

It would be ideal to have (3) in the real world, but we are not quite at that point yet. So either (1) or (2) should suffice. I will leave the choice to you although (2) might be good for practice since this most closely simulates what you would do if you were to store these locations in a database and you now know how to setup a backend endpoint using your knowledge from week 2 assignment :)